src/pages/editor/EditorInstitution.tsx

```diff
@@ -171,7 +172,7 @@ export const EditorInstitution = () => {
               <Grid container item xs={12}>
                 {institutionAdmins && (
                   <Grid item xs={12} sm={4} md={6} lg={4}>
-                    <Typography variant="h3" gutterBottom>
+                    <Typography variant="h3" component="h2" gutterBottom>
```

Comment:
Skal hver admin vises med en `<h2>`?

Dette er kanskje mer i tråd med det vi vil vise?

````
<h2>Administratorer</h2>
<ul>
  <li>Admin1</li>
  <li>Admin2</li>
  (...)
</ul>
````

Reply:
Dette gjøres jo kun for headeren? `institutionAdmins` filtrerer, og mappinga skjer under,

Reply:
Ja, stemmer 👍 Står igjen med om det skal se ut som h2 eller h3 da